airbyte-integrations/connectors/source-pipedrive/source_pipedrive/manifest.yaml

```diff
@@ -0,0 +1,339 @@
+version: "0.29.0"
+
+definitions:
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["data"]
+
+  selector_increment:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["data", "*", "data"]
+      
+  selector_users:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_path: ["data", "*", "data","*"]
```

Comment:
Why are you using this path?
![Uploading image.png…]()

Reply:
{
    "success": true,
    "data": [
        {
            "item": "user",
            "id": 18047894,
            "data": [
                {
                    "id": 18047894,
                    "name": "Aviraj G",
                    "default_currency": "INR",
                    "timezone_name": "Asia/Kolkata",
                    "timezone_offset": "+05:30",
                    "locale": "en_US",
                    "email": "***",
                    "phone": "***",
                    "created": "2023-08-07 05:15:57",
                    "modified": "2023-08-11 20:44:25",
                    "lang": 1,
                    "active_flag": true,
                    "is_admin": 1,
                    "last_login": "2023-08-11 20:44:25",
                    "signup_flow_variation": "",
                    "role_id": 1,
                    "has_created_company": false,
                    "icon_url": null,
                    "is_you": true
                }
            ]
        },
        {
            "item": "user",
            "id": 18105578,
            "data": [
                {
                    "id": 18105578,
                    "name": "Av",
                    "default_currency": "INR",
                    "timezone_name": "Asia/Kolkata",
                    "timezone_offset": "+05:30",
                    "locale": "en_US",
                    "email": "***",
                    "phone": null,
                    "created": "2023-08-12 09:23:56",
                    "modified": "2023-08-12 09:24:55",
                    "lang": 1,
                    "active_flag": true,
                    "is_admin": 0,
                    "last_login": "2023-08-12 09:24:55",
                    "signup_flow_variation": "",
                    "role_id": 1,
                    "has_created_company": false,
                    "icon_url": null,
                    "is_you": false
                }
            ]
        }
    ],
    "additional_data": {
        "since_timestamp": "2017-01-25 00:00:00",
        "last_timestamp_on_page": "2023-08-12 09:24:55",
        "pagination": {
            "start": 0,
            "limit": 2,
            "more_items_in_collection": false
        }
    }
}

Reply:
Api response for user that's why used this way

Reply:
else read response is throwing error